src/test/kotlin/study/PersonKoTest.kt

```diff
@@ -0,0 +1,30 @@
+package study
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.nulls.shouldBeNull
+import io.kotest.matchers.shouldBe
+
+class PersonKoTest : StringSpec({
+    "이름 붙인 인자" {
+        val person = Person(name = "김동인", age = 20, nickname = "eastperson")
+        person.name shouldBe "김동인"
+        person.age shouldBe 20
+        person.nickname shouldBe "eastperson"
+    }
+
+    "널 타입" {
+        val person = Person(name = "김동인", age = 20, nickname = null)
+        person.nickname.shouldBeNull()
+    }
+
+    "기본 인자" {
+        val person = Person(name = "김동인", age = 20)
+        person.name shouldBe person.nickname
+    }
```

Comment:
```suggestion
    "기본 인자" {
        val person = Person(name = "김동인", age = 20)
        person.name shouldBe "김동인"
    }
```
`person.nickname` 도 좋지만, 저는 상수로 작성된 코드가 더 잘 읽히더라구요! 
취향의 영역이므로 넘어가주셔도 됩니다. 😃

Reply:
앗 넵 ㅎㅎㅎㅎ 저도 상수로 선언하는 거 좋아합니다!! 1단계에는 정신없이 PR 여느라고 리팩토링을 못했네요 ㅋㅋㅋ